src/document-model/gen/header/creators.ts

```diff
@@ -17,40 +17,64 @@ import {
     SetAuthorWebsiteAction,
 } from './actions';
 
-export const setModelName = (input: SetModelNameInput) =>
+export const setModelName = (input: SetModelNameInput, skip = 0) =>
```

Comment:
Can you create a ticket for [powerhouse-inc/codegen](https://github.com/powerhouse-inc/codegen) to add this parameter to the code generation?